<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRL_ParametersManager</class>
 <widget class="QDialog" name="QRL_ParametersManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/parameters_icon.xpm</normaloff>:/icons/parameters_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" rowspan="4">
    <widget class="QListWidget" name="blockListWidget">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="batchCheckBox">
     <property name="text">
      <string>batch mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="downloadPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>download</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="uploadPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>upload</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2" colspan="3">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Block Parameters</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QTableWidget" name="parameterTableWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Options">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Blocks containing</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLineEdit" name="paramLineEdit">
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>IPARAM RPARAM</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="hideRadioButton">
            <property name="text">
             <string>should hide</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="showRadioButton">
            <property name="text">
             <string>should only visible</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QRadioButton" name="fineRadioButton">
            <property name="text">
             <string>are fine (Show all Parameter)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="savePushButton">
            <property name="text">
             <string>Save Parameter</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="loadPushButton">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Load Parameter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>(seperate different string with space)</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>blockListWidget</zorder>
  <zorder>batchCheckBox</zorder>
  <zorder>downloadPushButton</zorder>
  <zorder>uploadPushButton</zorder>
  <zorder></zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_ParametersManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
